demos/common/python/pipelines/async_pipeline.py

```diff
@@ -20,19 +20,36 @@
 
 
 class AsyncPipeline:
-    def __init__(self, ie, model, plugin_config, device='CPU', max_num_requests=1):
+    def __init__(self, ie, model=None, plugin_config={'CPU': 1}, device='CPU', max_num_requests=1, silent=False):
         self.model = model
-        self.logger = logging.getLogger()
+        self.logger = logging.getLogger() if not silent else lambda *x: None
 
-        self.logger.info('Loading network to {} plugin...'.format(device))
-        self.exec_net = ie.load_network(network=self.model.net, device_name=device,
-                                        config=plugin_config, num_requests=max_num_requests)
+        self.plugin_config = plugin_config
+        self.device = device
+        self.max_num_requests = max_num_requests
+        self.ie = ie
 
-        self.empty_requests = deque(self.exec_net.requests)
+        self.async_postprocessing = False
+
+        if self.model:
+            self.logger.info('Loading network to {} plugin...'.format(device))
```

Comment:
It fails with `AttributeError: 'function' object has no attribute 'info'`

Reply:
Done